.github/stale.yml

```diff
@@ -1,79 +0,0 @@
-# Configuration for probot-stale - https://github.com/probot/stale
-# Configuration options apply to both Issues and Pull Requests.
-# We configure those individually to match our workflow (see `pulls:` and `issues:`)
-
-pulls:
-  # Number of days of inactivity before a Pull Request becomes stale
-  daysUntilStale: 60
-
-  # Number of days of inactivity before a Pull Request with the stale label is closed.
-  # Set to false to disable. If disabled, Pull Request still need to be closed manually, but will remain marked as stale.
-  daysUntilClose: 21
-
-  # Comment to post when marking as stale. Set to `false` to disable
-  markComment: >
-    This pull request has been automatically marked as stale because it has not had recent activity.
-    Given the limited bandwidth of the team, it will be closed if no further activity occurs.
-    If you intend to work on this pull request, please ask the team to reopen the PR or push a new PR. 
-    Thank you for your contributions.
-
-  # Comment to post when closing a stale Pull Request.
-  closeComment: >
-    This pull request has been automatically closed due to inactivity.
-    If you are still interested in contributing this, please ensure that
-    it is rebased against the latest branch (usually `master`), all review
-    comments have been addressed and the build is passing.
-
-issues:
-  daysUntilStale: 365
-
-  # Number of days of inactivity before an Issue with the stale label is closed.
-  # Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
-  daysUntilClose: 21
-
-  # Comment to post when marking as stale. Set to `false` to disable
-  markComment: >
-    This issue has been automatically marked as stale because it has not had recent activity.
-    Given the limited bandwidth of the team, it will be automatically closed if no further
-    activity occurs. If you're interested in how we try to keep the backlog in a healthy state,
-    please read our [blog post on how we refine our backlog](https://blog.gradle.org/stale-issue-backlog).
-    If you feel this is something you could contribute, please have a look
-    at our [Contributor Guide](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md).
-    Thank you for your contribution.
-
-  # Comment to post when closing a stale Issue.
-  closeComment: >
-    This issue has been automatically closed due to inactivity. If you can reproduce this on a
-    recent version of Gradle or if you have a good use case for this feature, please feel free
-    to to let know so we can reopen the issue. Please try to provide steps to reproduce, a quick
-    explanation of your use case or a high-quality pull request.
-
-# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
-#onlyLabels: []
-
-# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
-exemptLabels:
-  - epic
```

Comment:
This hasn't been migrated, see `exempt-issue-labels`

Reply:
That's because I haven't found any epics.
Are those used anymore?

Reply:
Hah! I've looked into private's labels... I can see we are using them here.
I will move this.